.github/workflows/qa.yaml

```
@@ -40,6 +40,8 @@ jobs:
             - uses: bahmutov/npm-install@v1
             - run: npx stylelint assets/css
             - run: npx eslint assets/js
+            - run:
+                  npx tsc assets/js/**/*.ts --noEmit --skipLibCheck --esModuleInterop --pretty false
```

```suggestion
                  npx tsc --noEmit --skipLibCheck --esModuleInterop --pretty false
```

You shouldn't need to specify where are the typescript files, since it is a configuration from tsconfig.json

Why do you need those flags?

```
--noEmit 
--skipLibCheck 
--esModuleInterop
```

---

It is updated, but we need to keep `--noEmit` flag because it is not set in the `tsconfig.json` file as Webpack will throw `Error: TypeScript emitted no output ...` compilation error.

---

Updated: I have added a configuration to webpack encore, so we can remove this flag